src/status_im2/contexts/chat/messages/list/new_temp_view.cljs

```diff
@@ -103,6 +105,42 @@
           [content.deleted/deleted-message message-data context]
           [message/message-with-reactions message-data context keyboard-shown])]))])
```

Comment:
@OmarBasem, could you explain what's the idea behind the file `new_temp_view.cljs`? This is a pretty weirdly named file. Maybe I missed from other PR reviews, but do you know when this is going to be renamed/removed?

Reply:
That's because we are still keeping the old composer. Some style changes was done to `status-im2.contexts.chat.messages.list` which wouldn't work with the old composer. Hence that temporary file.